groups.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+// return true as soon as we find an item in set that is also in found
+func UserInGroup(set []string, found[]string) bool {
```

Comment:
this is more like an intersection of slices, not sure if the naming reflects that.

not linted